phpBB/adm/style/installer_header.html

```diff
@@ -48,7 +48,7 @@
 							<li<!-- IF l_block1.S_SELECTED --> id="activemenu"<!-- ENDIF -->><a href="{l_block1.U_TITLE}"><span>{l_block1.L_TITLE}</span></a></li>
 							<!-- END l_block1 -->
 							<!-- BEGIN l_block2 -->
-							<li<!-- IF l_block2.S_SELECTED --> id="activemenu"<!-- ENDIF -->><span<!-- IF l_block2.S_COMPLETE --> class="completed"<!-- ENDIF --> id="installer-stage-{l_block2.STAGE_NAME}">{l_block2.L_TITLE}</span></li>
+							<li<!-- IF l_block2.S_SELECTED --> id="activemenu"<!-- ENDIF -->><span<!-- IF l_block2.S_COMPLETE --> class="completed"<!-- ENDIF --> id="installer-stage-{l_block2.STAGE_NAME}"><!-- IF l_block2.IS_LINK --><a class="{l_block2.CLASS}" href="{l_block2.LINK}" task_index="{l_block2.TASK_INDEX}">{l_block2.L_TITLE}</a><!-- ELSE --> {l_block2.L_TITLE}<!-- ENDIF --> </span></li>
```

Comment:
Yes, I will change it. 
I have not finished the work with js, so I will change it further. I do not like how the js works at this point and would like to make more improvements to it